app/liquid_tags/jsitor_tag.rb

```diff
@@ -1,6 +1,6 @@
 class JsitorTag < LiquidTagBase
   PARTIAL = "liquids/jsitor".freeze
-  URL_REGEXP = %r{\A(https|http)://jsitor\.com/embed/\w+[-?a-zA-Z&]*\Z}
+  REGISTRY_REGEXP = %r{\A(https|http)://jsitor\.com/embed/\w+[-?a-zA-Z&]*\Z}
```

Comment:
Question around the `\w+[-?a-zA-Z&]*` versus /\A[\w&?-]+\Z/, are those for identifying IDs?

Reply:
I left the regex as it was, honestly. But I can do the shorthand version if you think it's better.

Reply:
My slight, mostly non-blocking preference, is for those to be in alignment.  But again, mostly non-blocking.  (And I recognize that this may be different from past reviews as I'm now thinking in detail about these).